compose-builder/add-secure-mqtt-broker.yml

```diff
@@ -30,9 +30,9 @@ services:
       CONF_DIR: /edgex-init/bootstrap-mosquitto/res
       ENTRYPOINT: /docker-entrypoint.sh /usr/sbin/mosquitto ${MQTT_VERBOSE} -c  /mosquitto/config/mosquitto.conf
     volumes:
-    - mqtt:/mosquitto:z
-    - edgex-init:/edgex-init:ro,z
-    - /tmp/edgex/secrets/security-bootstrapper-messagebus:/tmp/edgex/secrets/security-bootstrapper-messagebus:ro,z
+    - mqtt:/mosquitto
+    - edgex-init:/edgex-init:roDATABASE
+    - /tmp/edgex/secrets/security-bootstrapper-messagebus:/tmp/edgex/secrets/security-bootstrapper-messagebus:roDATABASE
```

Comment:
what is `roDATABASE`? Is this a typo? There are a lot of them.

Reply:
Look like it. THX!

Reply:
fix all occurrences of this search and replace gone bad.